include/circt/Dialect/Arc/Ops.h

```diff
@@ -22,4 +22,14 @@
 #define GET_OP_CLASSES
 #include "circt/Dialect/Arc/Arc.h.inc"
 
+namespace circt {
+namespace arc {
+
+/// Check if an operation causes logic to be split into separate arcs before and
+/// after it.
+bool isArcBreakingOp(Operation *op);
```

Comment:
I'm wondering, what's the motivation of moving this from a pass specific place to the general dialect implementation? As far as I can see, it is only ever used in the `ConvertToArcs` pass. Is there a potential need to call this in other dialects or passes?

Reply:
Yeah that's a good point. I think this is just a remnant from some old experiments. Moved this into the pass now.